src/components/SwapWidget/subcomponents/InfoSection/InfoSection.tsx

```diff
@@ -22,6 +22,7 @@ import {
 export type InfoSectionProps = {
   isApproving: boolean;
   isConnected: boolean;
+  isSelectedServer: boolean;
```

Comment:
```suggestion
  hasSelectedCustomServer: boolean;
```

Reply:
Added in change and pushed to this branch